src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdApi.java

```diff
@@ -253,29 +254,51 @@ public StartEventResponse startEvent(
         CcdAuthenticator authenticator,
         String jurisdiction,
         String caseTypeId,
-        String caseRef,
         String eventTypeId
     ) {
-        return caseRef == null
-            ? feignCcdApi.startForCaseworker(
-            authenticator.getUserToken(),
-            authenticator.getServiceToken(),
-            authenticator.getUserDetails().getId(),
-            jurisdiction,
-            caseTypeId,
-            eventTypeId
-        )
-            : feignCcdApi.startEventForCaseWorker(
+        return feignCcdApi.startForCaseworker(
             authenticator.getUserToken(),
             authenticator.getServiceToken(),
             authenticator.getUserDetails().getId(),
             jurisdiction,
             caseTypeId,
-            caseRef,
             eventTypeId
         );
     }
 
+    public StartEventResponse startEventForAttachScannedDocs(
+        CcdAuthenticator authenticator,
+        String jurisdiction,
+        String caseTypeId,
+        String caseRef,
+        String eventTypeId
+    ) {
+        try {
+            return feignCcdApi.startEventForCaseWorker(
+                authenticator.getUserToken(),
+                authenticator.getServiceToken(),
+                authenticator.getUserDetails().getId(),
+                jurisdiction,
+                caseTypeId,
+                caseRef,
+                eventTypeId
+            );
+        } catch (FeignException e) {
+            if (e.status() == 404) {
```

Comment:
We could specifically catch `FeignException.NotFound` instead, and `FeignException` in a subsequent catch. Up to you.

Reply:
updated in both start and submit events